index.html

```diff
@@ -185,49 +255,64 @@ <h3>Skills</h3>
       </section>
 
       <footer class="form-section">
-        <h2 class="form-desc">
-          I'm always interested in hearing about new projects, so if you'd like
-          to chat please get in touch.
-        </h2>
-
-        <form
-          action="https://formspree.io/f/xbjwrppl"
-          method="POST"
-          id="my-form"
-        >
-          <input
-            type="text"
-            id="name"
-            name="user_name"
-            placeholder="Full Name"
-            maxlength="30"
-            required
-          />
+        <div class="form-a">
+          <h2 class="form-desc">
+            I'm always interested in hearing about new projects, so if you'd
+            like to chat please get in touch.
+          </h2>
 
-          <input
-            type="email"
-            id="email"
-            placeholder="Email address"
-            name="user_email"
-            required
-          />
+          <form
+            action="https://formspree.io/f/xbjwrppl"
+            method="POST"
+            id="my-form"
+          >
+            <input
+              type="text"
+              id="name"
+              name="user_name"
+              placeholder="Full Name"
+              maxlength="30"
+              required
+            />
 
-          <textarea
-            name="message"
-            id="msg"
-            placeholder="Enter text here"
-            maxlength="500"
-            required
-          ></textarea>
+            <input
+              type="email"
+              id="email"
+              placeholder="Email address"
+              name="user_email"
+              required
+            />
 
-          <button type="submit" class="btn btn-center">Get In touch</button>
-        </form>
+            <textarea
+              name="message"
+              id="msg"
+              placeholder="Enter text here"
+              maxlength="500"
+              required
+            ></textarea>
 
+            <button type="submit" class="btn btn-center">Get In touch</button>
+          </form>
+        </div>
         <hr />
```

Comment:
- [ ] I think it could be a good idea to update this line width to take the entire viewport in order to match the Figma design and make your portfolio look even better :smiley: :raised_hands:

  Your design:
  ![image](https://user-images.githubusercontent.com/81522435/144722991-09f68239-4eba-42f8-8b59-be459b8a6ca2.png)

  Figma design:
  ![image](https://user-images.githubusercontent.com/81522435/144723000-8f0dc6a3-a620-4a2c-a716-592bf980da9e.png)